specification/containerservice/resource-manager/readme.python.md

```diff
@@ -53,6 +54,16 @@ Please also specify `--python-sdks-folder=<path to the root directory of your az
 python:
   namespace: azure.mgmt.containerservice.v2019_11_01
   output-folder: $(python-sdks-folder)/containerservice/azure-mgmt-containerservice/azure/mgmt/containerservice/v2019_11_01
+
```

Comment:
Missing ``` here to close the previous block

Reply:
thank you for catching those. It was lost in manual rebase confict resolution. :(